app/views/home/default.haml

```diff
@@ -4,6 +4,8 @@
       .row
         .col-md-8.text-center
           %h1= t("home.default.headline", pod_name: pod_name)
+          - if pod_name != "diaspora*"
+            %h2.clarification-header= t("home.default.clarify")
```

Comment:
Try `t('home.default.part_of_diaspora', diaspora_site_link: link_to(t('diaspora_site_link_text'), 'https://diasporafoundation.org/')).html_safe`

(I think this is correct, but please test it!)